Formula/swiftgen.rb

```diff
@@ -2,8 +2,8 @@ class Swiftgen < Formula
   desc "Swift code generator for assets, storyboards, Localizable.strings, …"
   homepage "https://github.com/SwiftGen/SwiftGen"
   url "https://github.com/SwiftGen/SwiftGen.git",
-      :tag => "5.2.1",
-      :revision => "21107b1de6180cf5644e512e67e4d60a0d115ac4"
+      :tag => "5.3.0-2",
```

Comment:
Github still lists 5.3.0 as latest release

Reply:
The tag 5.3.0-2 exists though https://github.com/SwiftGen/SwiftGen/releases/tag/5.3.0-2

As said in the commit message, that tag is temporary just to see if the changes made will properly fix that formula (which I'm not entirely convinced, especially about the error on High Sierra). Once we ensure we have found the right solution and the formula properly builds, I intend to delete tag 5.3.0-2 and create a real/official tag and squash the commits of this PR as if nothing bad happened.